package/MDAnalysis/visualization/RDKit.py

```diff
@@ -0,0 +1,212 @@
+            Parameters passed to RDKit for drawing the molecule
+        legend : str
+            Legend displayed on the image
+        **kwargs : object
+            Other parameters passed to :meth:`~rdkit.Chem.Draw.rdMolDraw2D.MolDraw2D.DrawMolecule`
+
+        Returns
+        -------
+        img : str
+            Raw PNG or SVG code
+        """
+        keep_3D = kwargs.pop("keep_3D", False)
+        mol = self._prepare_atomgroup_for_drawing(ag, keep_3D)
+        size = kwargs.pop("size", self.size)
+        useSVG = kwargs.pop("useSVG", self.useSVG)
+        drawer = MolDraw2DSVG if useSVG else MolDraw2DCairo
+        d2d = drawer(*size)
+        opts = kwargs.pop("drawOptions", self.drawOptions)
+        opts.prepareMolsBeforeDrawing = False
+        d2d.SetDrawOptions(opts)
+        d2d.DrawMolecule(mol, **kwargs)
+        d2d.FinishDrawing()
+        return d2d.GetDrawingText()
+
+    def atomgroup_to_gif(self, ag, output=None, legend="Frame {}",
+                         frame_duration=200, start=None, stop=None, step=None,
+                         **kwargs):
+        r"""Create a GIF from an AtomGroup
+
+        Parameters
+        ----------
+        ag : MDAnalysis.core.groups.AtomGroup
+            The AtomGroup to display
+        output : None or str
+            Either a path to save the gif (str), or ``None`` to display the GIF
+            inline
+        legend : str
+            Format string used for the legend of the GIF. ``{}`` will be
+            replaced by the frame number
+        frame_duration : int or list
+            Duration of each frame for the GIF
+        start : None or int
+            Start frame for the GIF
+        stop : None or int
+            End frame for the GIF
+        step : None or int
+            Skip 1 frame for every ``step`` frames in the trajectory
+        **kwargs : object
+            Other parameters used by :meth:`~atomgroup_to_image`
+        """
+        mol = self._prepare_atomgroup_for_drawing(ag, keep_3D=True)
+        if mol and hasattr(ag.universe, "trajectory"):
+            pngs = []
+            for ts in ag.universe.trajectory[start:stop:step]:
+                img = self.atomgroup_to_image(ag, keep_3D=True, useSVG=False,
+                                              legend=legend.format(ts.frame),
+                                              **kwargs)
+                pngs.append(img)
+            img, *imgs = [Image.open(BytesIO(png)) for png in pngs]
+            # write to file, or display if output is None
+            buffer = partial(open, output, "wb") if output else BytesIO
+            with buffer() as fp:
+                img.save(fp=fp, format='GIF', append_images=imgs,
+                         save_all=True, duration=frame_duration, loop=0)
+                if isinstance(fp, BytesIO):
+                    b64 = b64encode(fp.getvalue()).decode("ascii")
+                    display(HTML(f"<img src='data:image/gif;base64,{b64}' />"))
+
```

Comment:
```suggestion
else:
     raise ValueError("Creation of GIF requires a trajectory as part of the Universe")
```
Or something along those lines